### ISSUE TITLE
Case sensitivity?

### ISSUE THREAD

**Issue by eitsupi** [rhysd/actionlint#31]
Thank you for the excellent service.

When I ran the following workflow check on the web, I got an error that I think was caused by uppercase letters being identified as lowercase.
https://rhysd.github.io/actionlint/

```yml
name: Create new release
on:
  workflow_dispatch:

jobs:
  create_release:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - uses: ncipollo/release-action@v1
        with:
          allowUpdates: false
```

![image](https://user-images.githubusercontent.com/50911393/129475627-43662df7-bbab-4bdd-bed0-be3094c6664f.png)

**Comment by rhysd** [comment]
Thank you for reporting this. This is a bug of comparing input names.

**Comment by rhysd** [comment]
This was fixed and will be included in next v1.6.1 release which will approximately be created tomorrow. I already deployed the fix to the playground.

https://rhysd.github.io/actionlint/#eJxdjksOwyAMRPecwhdAUbtkFaln6DoixFFoKEbYhOuHfNRFl555T+Nov2jgldEKQsQKGQNaRkXRKIBKeZ0D1WHynKy4xSj1oZGPzp3ScAtHApBLZN1UKGOJUnRoBMtZsWDiiwLQUBjZgHXiKXLnFnQrFem35x8RnU8UAnX3jr6UfnvcYHvSy2J+F4BteH2n6dg2MNvAuAMvIUkq

**Comment by eitsupi** [comment]
Thank you for the quick fix!

**Comment by rhysd** [comment]
The fix was released at [v1.6.1](https://github.com/rhysd/actionlint/releases/tag/v1.6.1).
